src/rps-game.js

```diff
@@ -0,0 +1,112 @@
+import getThrow from './get-throw.js';
+
+
+const throwButton = document.getElementById('throw');
+const winImage = document.getElementById('winning-image');
+const message = document.getElementById('message');
+const winCount = document.getElementById('win-count');
+const lossCount = document.getElementById('loss-count');
+const drawCount = document.getElementById('draw-count');
+const userBetCount = document.getElementById('user-bet');
+const cpuBetCount = document.getElementById('cpu-bet');
+
+
+let wins = 0;
+let losses = 0;
+let draws = 0;
+let userBetAmount = 100;
+let cpuBetAmount = 100;
+
+let userMsg = userBetAmount;
+let cpuMsg = cpuBetAmount;
+
+userBetCount.textContent = userMsg;
+cpuBetCount.textCOntent = cpuMsg;
```

Comment:
typo here (textC0ntent vs textContent) - not sure how this is affecting your code though...